common.gradle

```diff
@@ -0,0 +1,26 @@
+ext {
+    theVersion = '0.22.0-SNAPSHOT'
```

Comment:
maybe name this `workspaceManagerVersion` or `serviceVersion`?

Reply:
I called it `wsmVersion` and am using it in all of the projects.